packages/palette/src/elements/Stepper/Stepper.tsx

```diff
@@ -20,37 +22,78 @@ export const Stepper: React.FC<StepperProps> = ({
   disableNavigation,
   initialTabIndex = 0,
   children,
+  fill,
   ...rest
 }) => {
+  const tokens = useThemeConfig({
+    v2: {
+      fill,
+      joinSeparator: <ChevronIcon fill="black30" width={12} mx={2} />,
+      inlineSeparator: null,
+      verticalAlignment: "center",
+      horizontalAlignment: "center",
+      checkAlignment: "left",
+      textVariant: "mediumText" as TextVariant,
+    },
+    v3: {
+      fill: true,
+      joinSeparator: null,
+      inlineSeparator: <ChevronIcon fill="black60" height={10} ml={1} />,
+      verticalAlignment: "center",
+      horizontalAlignment: "flex-start",
+      checkAlignment: "right",
+      textVariant: "sm" as TextVariant,
+    },
+  })
```

Comment:
Any reason this is inline with the coponent versus in its own `/tokens/v2|v3` file like the others?

Reply:
It's not the most compelling reason but I've just been doing that when they are CSS styles or actual variants which both get kind of long. All the ones that are like switches in values like this I've just been inlining since there tends to only be one or two. This one kinda just grew as I noticed some more differences between the two themes.